convert2rhel/utils.py

```diff
@@ -471,10 +471,17 @@ def download_pkg(
 
     output, ret_code = run_cmd_in_pty(cmd, print_output=False)
     if ret_code != 0:
-        loggerinst.warning(
-            "Couldn't download the %s package using yumdownloader.\n"
-            "Output from the yumdownloader call:\n%s" % (pkg, output)
-        )
+        loggerinst.warning("you are using an env that overides an inhibitor and 
+            you understand that if you continue you might lose the rollback."
+            \n"Output from the yumdownloader call:\n%s" % (pkg, output)
```

Comment:
You need to close the quotation marks on every physical line break in strings, otherwise, Python would understand you left the string open (by the initial quotation mark) and suddenly wanted to work on a new line, thus, creating an error... 

It's okay if you do something like this: 

```python
print("This gonna be a very very very long"
       " message for the user, which I will break a line for better readability.")
       
# Outputs
# This gonna be a very very very long message for the user, which I will break a line for better readability.
```

It didn't actually break a line in the message output, thus, you can create very long strings with easier reading without *actually* including a mandatory `\n` on your strings.

```suggestion
        loggerinst.warning("you are using an env that overides an inhibitor and" 
            " you understand that if you continue you might lose the rollback.\n"
            "Output from the yumdownloader call:\n%s" % (pkg, output)
```

Reply:
I have reworked this and merged this into the code!